Sources/web3swift/Tokens/ERC20/ERC20BaseProperties.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 /// Declares common properties of an [ERC-20](https://eips.ethereum.org/EIPS/eip-20) complient smart contract.
 /// Default implementation of access to these properties is declared in the extension of this protocol.
-public protocol ERC20BaseProperties: AnyObject {
+public protocol ERC20BaseProperties: Any {
```

Comment:
This one conformance should be literally useless in swift and can be dropped.

Reply:
This actually must be reverted back as making protocols conform to `AnyObject` limits them only to classes.
I haven't done yet review of this PR myself so probably will find some extra stuff to think about.
Thanks!

Reply:
Yep, this make sense, maybe this was the original intent of this line

Reply:
Fixed.